apps/projection/src/app/ui/card/card.component.html

```diff
@@ -1,29 +1,25 @@
-<div class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3" [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px"
-  />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px"
-  />
-
+<div
+  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
+  [class]="customClass">
+  <ng-container
+    [ngTemplateOutlet]="
+      templateView ? templateView : defaultView
+    "></ng-container>
```

Comment:
when you need to project static asset, use `ng-content`, use can use `select` to be more specific on what you want to project

Reply:
done